Policies/Kubernetes/fsgroup/rego/src_test.rego

```diff
@@ -0,0 +1,185 @@
+package k8sazurefsgroup
+
+test_input_fsgroup_allowed_all {
+    input := { "review": input_review_with_fsgroup, "parameters": input_parameters_runasany}
+    results := violation with input as input
+    count(results) == 0
+}
```

Comment:
nit: Add a test combination of input_review_with_securitycontext_no_fsgroup and input_parameters_runasany.